website/src/components/discourse/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import feedStyles from './styles.module.css';
+
+export default function DiscourseFeed({
+  order = 'latest_topic',
+  status = undefined,
+  after = undefined,
```

Comment:
Can we have relative periods here? I was thinking we'd do something like last 30 days for the help ones, and last 90 for discussions.

Reply:
Does this refer to the `after` property? If a different category is passed in other than `help` or `discussions` should we exclude the `after` date, or should we set a default number of days back?